tst/EnergyPlus/unit/FluidCoolers.unit.cc

```diff
@@ -356,10 +356,10 @@ TEST_F(EnergyPlusFixture, SizeFunctionTestWhenPlantSizingIndexIsZero)
     auto &thisFluidCooler = state->dataFluidCoolers->SimpleFluidCooler(FluidCoolerNum);
 
     state->dataPlnt->PlantLoop.allocate(FluidCoolerNum);
-    state->dataFluidCoolers->SimpleFluidCooler.allocate(FluidCoolerNum);
```

Comment:
Fix an existing test. When I tried to reuse it, I realized that the allocate after GetInput was resetting it to an empty (invalid / default initialized) FluidCooler